Metadata for grants page

Swader opened the pull request:
Grants page was missing header info and was showing up lowercase in sidemenu without a title on the page itself. This fixes the issue.

xlc commented on docs/lang/chinese/learn/interchain.md:
```diff
@@ -0,0 +1,39 @@
+
+在处理时，平行链B里的智能合约会执行该信息并且完成资产转移。
+
+现在校对人提交该区块给验证人并且核对信息是否处已经处理， 如果信息经已处理好和区块中其它方面是有效，验证人便会把该区块包括到中继链上。
```
```suggestion
然后校对人提交该区块给验证人，并且核对信息是否处已经处理。如果信息经已处理好和区块中其它方面是有效，验证人便会把该区块包括到中继链上。
```

xlc commented on docs/lang/chinese/learn/interchain.md:
```diff
@@ -0,0 +1,39 @@
+
+## 资源
+
+[ICMP方案](https://research.web3.foundation/en/latest/polkadot/ICMP/) - 完整的技术描述在Web3基金会研究网页
```
```suggestion
[ICMP方案](https://research.web3.foundation/en/latest/polkadot/ICMP/) - 在Web3基金会研究网页上完整的技术描述
```

Swader committed:
Metadata for grants page

Grants page was missing header info and was showing up lowercase in sidemenu without a title on the page itself. This fixes the issue.
